kubernetes_asyncio/client/rest.py

```diff
@@ -6,7 +6,7 @@
     No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)  # noqa: E501
 
     OpenAPI spec version: v1.10.1
-    
+
```

Comment:
Could you rollback this unnecessary change ?

Reply:
hah sure, that was just editor auto trimming trailing whitespace